src/PowerShellEditorServices/Services/PowerShellContext/PowerShellContextService.cs

```diff
@@ -2436,8 +2455,7 @@ private void OnBreakpointUpdated(object sender, BreakpointUpdatedEventArgs e)
 
         private void ConfigureRunspaceCapabilities(RunspaceDetails runspaceDetails)
         {
-            // TODO: Bring this back
-            //DscBreakpointCapability.CheckForCapability(this.CurrentRunspace, this, this.logger);
+            DscBreakpointCapability.CheckForCapability(this.CurrentRunspace, this, this.logger);
```

Comment:
Do we know that this works now? I don't actually remember if it broke at some point or more generally why it's commented, just curious.

Reply:
It was commented out by me in the omnisharp transition because at the time I didn't port over the debug adapter yet.

Now we have that and this is good to uncomment.

Not sure if it actually works, but I'm putting the code back where I found it.